demo/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:8.16.0-stretch AS texture
+
+USER node
+ENV HOME=/home/node
+
+ENV PROJECT_FOLDER=${HOME}/sciencebeam-texture
+
+RUN mkdir ${PROJECT_FOLDER}
+WORKDIR ${PROJECT_FOLDER}
+
+COPY --chown=node:node package.json package-lock.json ${PROJECT_FOLDER}/
+RUN npm ci
+
+COPY --chown=node:node . ${PROJECT_FOLDER}/
```

Comment:
do the files need to be modified at runtime? Nowadays I tend to go for `root` ownership so that they are read only.

Reply:
I wouldn't think the files need to be modified after the build is complete. i.e. after `npm ci` and `npm run build`. Running those commands as `node` might be slightly more secure in case there is a rough dependency? I suppose ideally the server would be started with another user having limited write permissions if any.

Reply:
It's all academical at the moment, not a problem.